std-bits/base/src/main/java/org/enso/base/enso_cloud/EnsoSecretHelper.java

```diff
@@ -28,16 +28,20 @@
 public final class EnsoSecretHelper extends SecretValueResolver {
   private static EnsoHTTPResponseCache cache;
 
-  /** Gets a JDBC connection resolving EnsoKeyValuePair into the properties. */
+  /**
+   * Gets a JDBC connection resolving EnsoKeyValuePair into the properties.
+   *
+   * @param properties properties in the form of {@code List<Pair<String, HideableValue>>}
```

Comment:
Why do we think `List<Pair<String, HideableValue>>` is better than simple `Map<String, HideableValue>`? It would be easier for the _dual JVM mode_ to avoid dealing with `Pair`...

Reply:
- after the rewrite to `Map` at https://github.com/enso-org/enso/commit/9a980fe5d37d4aeb711d64f8a5d12de9a51081ba
- I know why the code uses `List<Pair>` instead
- without that [Multiple.content.types.in.the.header.list.are.respected](https://github.com/enso-org/enso/blob/7818078183b40e79a63281af3c423feb624ef94b/test/Base_Tests/src/Network/Http_Spec.enso#L508) fails
- https://github.com/enso-org/enso/blob/7818078183b40e79a63281af3c423feb624ef94b/test/Base_Tests/src/Network/Http_Spec.enso#L509
- setting headers to `headers=[Header.content_type "application/json", Header.content_type "text/plain"]`